src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java

```diff
@@ -100,25 +105,42 @@
         Map<String, Object> modifiedCaseData = new HashMap<>(transformedCaseData);
 
         modifiedCaseData.put(PAPER_APPLICATION, YES_VALUE);
+        modifiedCaseData.put(APPLICANT_REPRESENTED, getValueForIsRepresented(modifiedCaseData));
+        modifiedCaseData.put(SOLICITOR_AGREE_TO_RECEIVE_EMAILS, getSolicitorAgreeToReceiveEmailsField(modifiedCaseData));
+        modifiedCaseData.put(RESPONDENT_REPRESENTED, getRespondentRepresentedField(modifiedCaseData));
 
         // If OrderForChildren is populated then set orderForChildrenQuestion1 to Yes
         if (StringUtils.isNotEmpty((String) modifiedCaseData.get("natureOfApplication5b"))) {
             modifiedCaseData.put("orderForChildrenQuestion1", YES_VALUE);
         }
 
+        return modifiedCaseData;
+    }
+
+    private String getValueForIsRepresented(Map<String, Object> modifiedCaseData) {
+        // If FR_APPLICANT_REPRESENTED_3 was chosen then set APPLICANT_REPRESENTED to Yes
+        String applicantRepresentedPaperValue = nullToEmpty(modifiedCaseData.get(APPLICANT_REPRESENTED_PAPER));
+        if (applicantRepresentedPaperValue.equalsIgnoreCase(FR_APPLICANT_REPRESENTED_3)) {
+            return YES_VALUE;
+        } else {
+            return NO_VALUE;
+        }
+    }
+
+    private String getSolicitorAgreeToReceiveEmailsField(Map<String, Object> modifiedCaseData) {
         if (StringUtils.isNotEmpty((String) modifiedCaseData.get(SOLICITOR_EMAIL))) {
```

Comment:
this looks like a good candidate for an util method - FormAToCaseTransformer.java is becoming faaar toooooo long:

```
String getYesOrNoValue(Map<String, Object> caseData, String field)
```